packages/devtools_app/lib/src/shared/tree.dart

```diff
@@ -35,7 +35,7 @@ class TreeView<T extends TreeNode<T>> extends StatefulWidget {
   /// Invoked when a tree node is selected. If [onItemExpanded] is not
   /// provided, this method will also be called when the expand button is
   /// tapped.
-  final FutureOr<void> Function(T) onItemSelected;
+  final FutureOr<void> Function(T)? onItemSelected;
```

Comment:
nit: instead of making this nullable why not just provide a NOOP implementation of this for now for your use case. I imagine you will want to provide a onItemSelected callback at some point.

Reply:
I made this `nullable` because I am using `onItemExpanded` instead. The doc for this method says: "If [onItemExpanded] is not provided, this method will also be called when the expand button is tapped" which makes it sound like it should be nullable. I also see we are passing it as a no-op in other parts of DevTools ( I can remove that if we want to keep it nullable): 
 https://github.com/flutter/devtools/blob/ec967fc55ff06a66095117214d2980166bd44bd4/packages/devtools_app/lib/src/screens/vm_developer/object_inspector/class_hierarchy_explorer.dart#L39

Reply:
Switched to removing the places where we are passing this as a no-op